src/MCPClient/lib/clientScripts/getAipStorageLocations.py

```diff
@@ -20,6 +20,7 @@ def get_aip_storage_locations(purpose):
     choices = {}
     for storage_dir in storage_directories:
         choices[storage_dir['description']] = storage_dir['resource_uri']
+    choices['Default location'] = '/api/v2/location/default/{}/'.format(purpose)
```

Comment:
Optionally I could check if `/api/v2/location/default/AS/` returns a real resource or a 404 (i.e. no default is defined in SS) and omit.